src/client/components/Input/Input.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './Input.css';
+
+export default function Input(prop) {
```

Comment:
```suggestion
export default function Input(props) {
```
It is common to name this `props`.